templates/service.epp

```diff
@@ -0,0 +1,11 @@
+<%- | String $metrics_command, String $service | -%>
+[Unit]
+Description=Oneshot service to capture PE metrics
```

Comment:
Maybe use Puppet metrics instead of PE?

Reply:
Will do.  One thing that I want to investigate is using `systemd` specifiers to only have one service for capturing metrics and one for tidying.  Then the description could say `Service to capture %i metrics`.